plugin.video.svtplay/resources/lib/logging.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import absolute_import,unicode_literals
+import xbmc # pylint: disable=import-error
+
+def log(msg, loglevel=xbmc.LOGDEBUG):
+    log_str = "[plugin.video.svtplay] {}".format(msg)
+    try:
```

Comment:
In python3 you can now use fstrings :)

Reply:
Yeah, but are those Py2 compatible? 🤔 @enen92

Reply:
Nop :) it was just a tip

Reply:
Ah, OK. Thanks! 👍